classes/ImageManager.php

```diff
@@ -246,20 +248,24 @@ public static function resize(
             $sourceHeight = $tmpHeight;
         }
 
-        // If PS_IMAGE_QUALITY is activated, the generated image will be a PNG with .jpg as a file extension.
-        // This allow for higher quality and for transparency. JPG source files will also benefit from a higher quality
-        // because JPG reencoding by GD, even with max quality setting, degrades the image.
-        if (Configuration::get('PS_IMAGE_QUALITY') == 'png_all'
-            || (Configuration::get('PS_IMAGE_QUALITY') == 'png' && $type == IMAGETYPE_PNG) && !$forceType) {
-            $fileType = 'png';
-        }
+        $isMultipleImageFormatFeatureActive = FeatureFlag::isEnabled(FeatureFlagSettings::FEATURE_FLAG_MULTIPLE_IMAGE_FORMAT);
```

Comment:
Can't you use `SymfonyContainer::getInstance()->get('prestashop.core.admin.feature_flag.repository')->isEnabled(FeatureFlagSettings::FEATURE_FLAG_MULTIPLE_IMAGE_FORMAT)` instead?
```suggestion
        $isMultipleImageFormatFeatureActive = SymfonyContainer::getInstance()->get('prestashop.core.admin.feature_flag.repository')->isEnabled(FeatureFlagSettings::FEATURE_FLAG_MULTIPLE_IMAGE_FORMAT);
```

Reply:
I applied your suggestion in `AdminImageController` but I reverted to previous implementation in `ImageManager`:

it breaks a lot of legacy controller integration tests (controllers that use this class) and I'm not sure how to fix them using SymfonyContainer's static `getInstance` method...